lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+
+.nx-tabs {
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
+
+.nx-modal-content--tabs {
```

Comment:
This class should be documented somewhere.  And should it be defined here or in `NxModal.scss`?

Reply:
> This class should be documented somewhere. And should it be defined here or in `NxModal.scss`?

Documented here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/7109584281b08533c26c5c541b17e319dc56a5bc

I moved the modifier over to `nx-modal.scss` mostly because I think that's the most consistent with what we do elsewhere: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/c9b9b82706dd0ab322d28c31d084e138fe7f3e61

Reply:
> This class should be documented somewhere. And should it be defined here or in `NxModal.scss`?

Documented here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/7109584281b08533c26c5c541b17e319dc56a5bc

And moved here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/c9b9b82706dd0ab322d28c31d084e138fe7f3e61